src/sys/sockets.nim

```diff
@@ -883,3 +884,49 @@ proc accept*(l: AsyncListener[TCP]): tuple[conn: AsyncConn[TCP], remote: IPEndpo
 proc localEndpoint*(l: AsyncListener[TCP] | Listener[TCP]): IPEndpoint {.raises: [OSError].} =
   ## Obtain the local endpoint of `l`.
   tcpLocalEndpoint()
+
+proc connectUnix*(path: string): Conn[Unix]
+                {.raises: [OSError].} =
+  ## Create a connection to the Unix socket at `path`.
```

Comment:
Can you add a "Platform specific details" block about how this API does not support Windows at the moment and that the length limit varies between operating systems?

Then please replicate that block to all Unix sockets functions docs that take a path.